css/properties.json

```diff
@@ -4712,6 +4640,49 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "offset": {
+    "syntax": "&lt;'offset-path'&gt; &amp;&amp; &lt;'offset-distance'&gt; &amp;&amp; &lt;'offset-rotation'&gt; &amp;&amp; &lt;'offset-anchor'&gt;",
```

Comment:
The syntax seems wrong. The [current definition](https://drafts.fxtf.org/motion-1/#propdef-offset) looks like this:

    [ <offset-position>? [ <offset-path> [ <offset-distance> || <offset-rotate> ]? ]? ]! [ / <offset-anchor> ]?

Sebastian